benchmark/fio_benchmark_setup.yml

```diff
@@ -1,5 +1,42 @@
 ---
-- hosts:
+- name: Build FIO
+  hosts: localhost,benchmark_hosts
```

Comment:
one question, do we want to run this against localhost and benchmark_hosts?

Reply:
For future ref: Yea the results are currently fetched back to the ansible client and then graphed.  That could change tho.  We could have the clients graph and then pull back the logs+graphs.